### DISCUSSION TITLE
React-spring can't be remove for XYChart ?

### DISCUSSION THREAD

**Discussion by LoiKos** [airbnb/visx#1203] [Q&A] [answered]
I try to use XYChart provide without installing `react-spring` and try using `Axis` component which shouldn't be animated but it couldn't compile cause of react-spring missing.
Version i use for visx: 

    "@visx/axis": "^1.10.0",
    "@visx/event": "^1.7.0",
    "@visx/group": "^1.7.0",
    "@visx/mock-data": "^1.7.0",
    "@visx/pattern": "^1.7.0",
    "@visx/scale": "^1.7.0",
    "@visx/shape": "^1.8.0",
    "@visx/stats": "^1.7.0",
    "@visx/tooltip": "^1.7.2",
    "@visx/xychart": "^1.10.0",

I use other package of visx elsewhere and they work fine without `react-spring` i just have trouble with xychart

**Comment by williaster** [comment, marked as answer]
Hey @LoiKos thanks for raising this issue. Currently `react-spring` is listed as a [`peerDependency` of `@visx/xychart`](https://github.com/airbnb/visx/blob/master/packages/visx-xychart/package.json#L40). I think this isn't necessarily ideal particularly if you don't use the animated components but I haven't figured out a great way to create two indexes for the package: one for non-animated components (no need to install `react-spring`) and one for animated components.

**Reply by LoiKos** [reply]
all rights that fine, i just included react-spring for now. I don't know much on bundling/releasing package so i can't help you on this sorry.
